content/en/observability_pipelines/_index.md

```diff
@@ -46,6 +46,9 @@ The Datadog UI provides a control plane to manage your Observability Pipelines W
 
 1. [Set up the Observability Pipelines Worker][1].
 2. [Create pipelines to collect, transform and route your data][2].
+3. Deep dive into recommendations for deploying and scaling multipe Observability Pipelines Workers:
```

Comment:
```suggestion
3. Deep dive into recommendations for deploying and scaling multiple Observability Pipelines Workers:
```

Spelling.

Reply:
But this also reads kind of funny to me. On one hand I think `Dive deep` sounds better to me, but I am not sure if that is grammatically correct or not. But maybe side step it altogether with an alternative phrasing such as:

"Discover how to deploy Observability Pipelines at production scale"

Thoughts @barieom ?

Reply:
That suggestion sounds good to me!

Reply:
Agree, I like your suggestion @ckelner